src/azure-cli/azure/cli/command_modules/botservice/_params.py

```diff
@@ -163,6 +163,16 @@ def load_arguments(self, _):
         c.argument('site_name', options_list=['-s', '--site-name'], help='Name of the Directline channel site.')
         c.argument('is_v1_disabled', options_list=['--disablev1'], help='If true, v1 protocol will be disabled on the channel', arg_type=get_three_state_flag())
         c.argument('is_v3_disabled', options_list=['--disablev3'], help='If true, v3 protocol will be disabled on the channel.', arg_type=get_three_state_flag())
+        c.argument('is_secure_site_enabled', help='If true, enables enhanced authentication features. Must be true for --trusted-origins parameters to work.', arg_type=get_three_state_flag())
```

Comment:
Have you placed these new added arguments for `az bot directline create` in release notes?